data/tlg2733/tlg001/tlg2733.tlg001.1st1K-grc1.xml

```diff
@@ -51,8 +50,8 @@
                     <monogr>
                         <title>The ecclesiastical history of Evagrius with the scholia</title>
                         <author>Evagrius, Scholasticus</author>
```

Comment:
Also perhaps an `xml:lang="lat"` on this `<author>` element.
Also below (I can't comment on that line because Github won't let me) in `<langUsage>` a `<language>` element for Latin should be added.